docs/CONTRIBUTING.md

```diff
@@ -77,6 +77,9 @@ At a high level, [the process for proposing changes](https://guides.github.com/i
 
 `script/cibuild`
 
+## Publishing Gem
+In order to publish the Gem bump the version in *this file*, commit, create a tag with the new version number, and finally push the commit and tag to the repo.
+
```

Comment:
"…in this file" — this file is CONTRIBUTING.md — wasn't there supposed to be a link? Probably to the gemspec?

Anyways the workflow isn't tied to tags, but to releases. So the instructions don't seem correct?